kinto_http/session.py

```diff
@@ -43,6 +49,7 @@ def __init__(self, server_url, auth=None, retry=0, retry_after=None):
         self.auth = auth
         self.nb_retry = retry
         self.retry_after = retry_after
+        self.headers = 'kinto-http.py/{} requests/{} python/{}'.format(kinto_http_version , requests_version, python_version)
```

Comment:
This does not have to be a class member, you can define a variable on the module (after python version). Also it should be called *user agent* and not *headers*